mvp/2023/mvp-enriched.json

```diff
@@ -535,10 +535,7 @@
           "M0005",
           "M0003",
           "M0004"
-        ],
```

Comment:
These were showing up on IA measures, left over from the ingestion issue we had for spec urls.